### ISSUE TITLE
Issue with Vim Tmux Navigator?

### ISSUE THREAD

**Issue by thewatts** [tmate-io/tmate#33]
There's an issue using `tmate` and [Vim Tmux Navigator](https://github.com/christoomey/vim-tmux-navigator) together.

It looks like the issue is because of mismatching versions of `tmate` and `tmux`.
[More info here](https://github.com/christoomey/vim-tmux-navigator#it-doesnt-work-in-tmate) and [here](https://github.com/christoomey/vim-tmux-navigator/issues/27).

**Comment by murphyrandle** [comment]
Youch, I'm hoping for this also, but I don't think I'll be much help. @nviennot Thanks for tmate!

**Comment by nviennot** [comment]
If you put this in your .bashrc / .zshrc:

```bash
if [[ $TMUX =~ tmate ]]; then alias tmux=tmate; fi 
```
does it help?

**Comment by murphyrandle** [comment]
I'm afraid it wouldn't help me much, the thing that chafes me is that I can't use `pane_current_path` in tmate to have new splits be in the same directory as the originating split.

It doesn't seem like aliasing tmux to tmate would solve that at all, as it's a tmux api incompatibility. Right?

**Comment by nviennot** [comment]
oh I see. So for the directory split issue, you can have a quick hack:

if you have in your `~/.tmux.conf`:

```
bind -n M-\ split-window -h -c '#{pane_current_path}'
bind -n M-- split-window -v -c '#{pane_current_path}'
```

put in your '~/.tmate.conf`:

```
bind -n M-\ split-window -h
bind -n M-- split-window -v
```

That is, you just remove the `-c ... ` argument.

**Comment by murphyrandle** [comment]
Well, I feel silly. For now then, I don't feel an urgent need for 1.9 :) I've downgraded tmux to 1.8
Thanks for your help!

**Comment by nviennot** [comment]
updating tmate to tmux 1.9 is really on my list, but I'm being a little conservative because I'm not sure what could break and all that. I don't want to introduce regressions, because I know I won't have the time to deal with them. Sorry about that :)

**Comment by murphyrandle** [comment]
Not a problem. Thank you! 

—

**Comment by pmccarren** [comment]
Any plans for adopting tmux 1.9a?

**Comment by nviennot** [comment]
Yes! But no time yet

**Comment by jondkinney** [comment]
+1 if only I knew C I'd love to help out :/

**Comment by daviddavis** [comment]
:+1: is this the issue being used to track 1.9 support?

**Comment by christoomey** [comment]
There is a new PR in work on the vim-tmux-navigator repo that automatically detects `tmate` based on [this snippet](https://github.com/nviennot/tmate/issues/33#issuecomment-69130656) from @nviennot.

You can see the PR here: https://github.com/christoomey/vim-tmux-navigator/pull/89

P.S. Thanks a ton for tmate @nviennot, it's great!

**Comment by nviennot** [comment]
I'll update to the latest tmux soon :)

**Comment by christoomey** [comment]
Follow up for anyone using Vim Tmux Navigator. I've gone ahead and merged the fix (christoomey/vim-tmux-navigator#89) so if you pull locally all should be set. This works with the current `tmate`. I'll leave to @nviennot to close, but I believe this issue is resolved.

**Comment by nviennot** [comment]
tmate has been upgraded to latest version of tmux.

Run brew install --HEAD tmate if you use homebrew. Install from sources (git HEAD) if not.
